ocean_lib/ocean/ocean_assets.py

```diff
@@ -224,7 +224,7 @@ def create(self, metadata: dict, publisher_wallet: Wallet,
         logger.debug('Encrypting content urls in the metadata.')
 
         publisher_signature = self._data_provider.sign_message(publisher_wallet, asset.asset_id, self._config)
-        encrypt_endpoint = self._data_provider.build_encrypt_endpoint(provider_uri)
+        encrypt_endpoint_method, encrypt_endpoint = self._data_provider.build_encrypt_endpoint(provider_uri)
```

Comment:
please use `_` here and in other places where you don't plan on using the variable name in further lines. This way we can avoid warnings about unused variables.